src/constants/wallets.ts

```diff
@@ -105,7 +107,12 @@ const detectRabby = (): WalletReadyState => {
 
 const detectKrystal = (): WalletReadyState => {
   if (getIsKrystalWallet()) return WalletReadyState.Installed
-  return WalletReadyState.NotDetected
+  return WalletReadyState.Unsupported
+}
+
+const detectKrystalWC = (): WalletReadyState => {
```

Comment:
detectKrystalWC vs detectKrystal: no difference

Reply:
renamed for easier to understand